Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM frolvlad/alpine-glibc:alpine-3.10
+
+# much of image code ripped from
+# https://github.com/Docker-Hub-frolvlad/docker-alpine-miniconda3
+COPY BASE_IMAGE_LICENSE /
+
+LABEL maintainer="conda-forge (@conda-forge/core)"
+
+ENV LANG en_US.UTF-8
+
+ARG CONDA_VERSION="4.7.12.1"
+ARG CONDA_MD5="81c773ff87af5cfac79ab862942ab6b3"
+ARG CONDA_DIR="/opt/conda"
+
+ENV PATH="$CONDA_DIR/bin:$PATH"
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# make sure the install below is not cached by docker
+ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
+
+# Install conda
+RUN echo "**** install dev packages ****" && \
+    apk add --no-cache bash ca-certificates wget && \
+    \
+    echo "**** get Miniconda ****" && \
+    mkdir -p "$CONDA_DIR" && \
+    wget "http://repo.continuum.io/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh" -O miniconda.sh && \
```

Comment:
This could use miniforge

Reply:
link?

Reply:
https://github.com/conda-forge/miniforge

Reply:
ah cool - i'm gonna punt on this for now but will come back later

Reply:
OK